Now it's mac in unit-tests:

>   make[4]: *** No rule to make target `/Users/runner/.cargo/git/checkouts/mozjs-fa11ffc7d4f1cc2d/8ac1958/mozjs-sys/mozjs/intl/icu/source/common/servnotf.cpp', needed by `servnotf.o'.  Stop.

--- a/components/script/dom/console.rs
+++ b/components/script/dom/console.rs
@@ -27,8 +27,8 @@ const MAX_LOG_DEPTH: usize = 10;
 /// The maximum elements in an object logged by console methods.
 const MAX_LOG_CHILDREN: usize = 15;
 
-// https://developer.mozilla.org/en-US/docs/Web/API/Console
-pub struct Console(());
+/// <https://developer.mozilla.org/en-US/docs/Web/API/Console>
+pub struct Console;
 
 impl Console {
     #[allow(unsafe_code)]
@@ -260,16 +260,11 @@ impl consoleMethods<crate::DomTypeHolder> for Console {
     }
 
     // https://developer.mozilla.org/en-US/docs/Web/API/Console/assert
-    fn Assert(_cx: JSContext, global: &GlobalScope, condition: bool, message: HandleValue) {
+    fn Assert(_cx: JSContext, global: &GlobalScope, condition: bool, messages: Vec<HandleValue>) {
         if !condition {
-            let message = if message.is_undefined() {
-                DOMString::from("no message")
-            } else {
-                stringify_handle_value(message)
-            };
-            let message = DOMString::from(format!("Assertion failed: {}", message));
-            console_message(global, message, LogLevel::Error)
-        };
+            let message = DOMString::from(format!("Assertion failed: {}", stringify_handle_values(messages)));
+            console_message(global, message, LogLevel::Error);
+        }
     }
 
     // https://console.spec.whatwg.org/#time

--- a/components/script/dom/webidls/Console.webidl
+++ b/components/script/dom/webidls/Console.webidl
@@ -8,7 +8,7 @@
  Exposed=*]
 namespace console {
   // Logging
-  undefined assert(boolean condition, optional any message);
+  undefined assert(optional boolean condition = false, any... data);
   undefined clear();
   undefined debug(any... messages);
   undefined error(any... messages);

--- a/components/script/dom/webidls/Console.webidl
+++ b/components/script/dom/webidls/Console.webidl
@@ -5,16 +5,20 @@
 // https://console.spec.whatwg.org/
 
 [ClassString="Console",
- Exposed=(Window,Worker,Worklet)]
+ Exposed=*]
 namespace console {
   // Logging
-  undefined log(any... messages);
-  undefined debug(any... messages);
-  undefined info(any... messages);
-  undefined warn(any... messages);
-  undefined error(any... messages);
   undefined assert(boolean condition, optional any message);
   undefined clear();
+  undefined debug(any... messages);
+  undefined error(any... messages);
+  undefined info(any... messages);
+  undefined log(any... messages);
+  // undefined table(optional any tabularData, optional sequence<DOMString> properties);
+  // undefined trace(any... data);
+  undefined warn(any... messages);
+  // undefined dir(optional any item, optional object? options);
+  // undefined dirxml(any... data);
 
   // Counting
   undefined count(optional DOMString label = "default");

--- a/tests/wpt/meta/console/idlharness.any.js.ini
+++ b/tests/wpt/meta/console/idlharness.any.js.ini
@@ -1,11 +1,5 @@
 [idlharness.any.html]
-  [console namespace: operation assert(optional boolean, any...)]
-    expected: FAIL
-
   [console namespace: operation table(optional any, optional sequence<DOMString>)]
-    expected: FAIL
-
-  [console namespace: operation trace(any...)]
     expected: FAIL
 
   [console namespace: operation dir(optional any, optional object?)]
@@ -16,13 +10,7 @@
 
 
 [idlharness.any.worker.html]
-  [console namespace: operation assert(optional boolean, any...)]
-    expected: FAIL
-
   [console namespace: operation table(optional any, optional sequence<DOMString>)]
-    expected: FAIL
-
-  [console namespace: operation trace(any...)]
     expected: FAIL
 
   [console namespace: operation dir(optional any, optional object?)]

